static/includes/apps/AppsInstallWizardSettings.md

```diff
@@ -27,14 +29,46 @@ All applications include these basic setting sections:
   Some network configuration settings include the option to add a certificate. Create the certificate authority and certificate before using the installation wizard if using a certificate is required for the application.
 
 * **Storage Configuration** shows options to configure storage for the application.
-  Storage options include using the default ixVolume setting that adds a storage volume under the **ix-applications** dataset, host path where you select existing dataset(s) to use, or in some cases the SMB share option where you configure a share for the application to use.
-  The **Add** button allows you to configure additional storage volumes for the application to use in addition to the main storage volume (dataset).
+  Storage configuration can include the primary data mount volume, a configuration volume, postgres volumes, and an option to add additional storage volumes.
+  The primary mount volumes have two options:
+* **ixVolume** that creates a storage volume inside the hidden **ix-apps** dataset. This is the default setting.
+* **Host Path** that allows you to select an existing dataset created for the application. Shows additional fields to select the path to the dataset and add the mount point.
 
-  If the application requires specific datasets, configure these before using the installation wizard.
+  ixVolumes are not recommended for permanent storage volumes, they are intended for use as rapid storage for a test deployment of the container.
+  We recommend adding datasets and configuring the container storage volumes with the host path option.
 
-  {{< include file="/static/includes/FileExplorerFolderIcons.md" >}}
+  Host path add existing dataset(s) as the storage volumes. You must configure the datasets before beginning the app installation using the wizard.
+
+  {{< hint type="information" title="ix-apps Dataset" >}}
```

Comment:
This hint type is rendering oddly in local build and not playing well with dark mode - lets change to one of the standard types (suggestion is a blue note box)
```suggestion
  {{< hint type="note" title="ix-apps Dataset" >}}
```